for/matProp.for

```diff
@@ -24,6 +24,7 @@ Module matProp_Mod
     Double Precision :: cl                                           ! Opt. fiber nonlinearity (defaults to zero, which is no fiber nonlinearity)
     Double Precision :: mu                                               ! Opt. Friction. Defaults to zero
     Double Precision :: es(4), gs(4)
+    Double Precision :: a6, b2, n, A                                       ! Specified parameters for schaefer theory
```

Comment:
I'm concerned about the very generic names n and A. Consider creating more descriptive names to avoid confusion.